src/pretix/settings.py

```diff
@@ -735,6 +736,31 @@
     ('pretix.plugins.banktransfer.*', {'queue': 'background'}),
 ],)
 
+CELERY_BEAT_SCHEDULE = {
```

Comment:
Tasks which run periodically, better implemented with [systemd timer](https://wiki.archlinux.org/title/Systemd/Timers). Benefit:

- Beside the time point which you scheduled, you can run the task any time, (when you need to test, or there is a need to run task one more time) with the command `sudo systemctl start task-name.service`.

- Easily to temporary disable the schedule, for example when you found that there is bug and you want to disable the task until you fix, by `sudo systemctl stop task-name.timer`.

- Logs are collected by `journald` (part of systemd), with rich feature to filter and view logs.